pkg/events/example_basic_test.go

```diff
@@ -32,3 +33,36 @@ func Example_basic() {
 		log.Fatal("error posting custom event:", err)
 	}
 }
+
+func Example_batch() {
```

Comment:
Can you put this in its own file (`example_batch_test.go`?) so the godoc will render it as a separate use case?

Reply:
yeah good call.